google-home-community.groovy

```diff
@@ -3488,6 +3533,22 @@ private deviceStateForTrait_StartStop(deviceTrait, device) {
     return deviceState
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private deviceStateForTrait_StatusReport(deviceTrait, device) {
+    def statusReport = []
+    (0..<deviceTrait.numStatusTypes).each { i ->
+        statusReport << [
+            deviceTarget: deviceTrait.statusTypes[i].deviceTargetAttribute,
```

Comment:
This doesn't really make sense to me.  `deviceTarget` is supposed to be the ID of a device (i.e. the ID of this device, or the ID of another device if aggregating statuses).